deploy/csi-azuredisk-controller.yaml

```diff
@@ -31,7 +31,7 @@ spec:
           effect: "NoSchedule"
```

Comment:
Do we also need to make these changes to the latest-v2 yaml files?

Reply:
Done